### PR TITLE
Update Appveyor dependencies package, version 17.05:

### PR DESCRIPTION
**Pull request by ximinez**
* boost 1.70, including b2.exe and bjam.exe.
* Remove long obsolete packages.

### CODE REVIEW

File: appveyor/README.md [hunk 1]
```diff
@@ -44,3 +16,18 @@ optimize downloads and caching for Appveyor.com CI builds.
     * protoc 2.5.1
     * Boost 1.67
     * OpenSSL 1.0.2j (64-bit) from http://slproweb.com/products/Win32OpenSSL.html
+* Version 17.05 splits the single zip file into several to stay under Github's
+  file size limits.
+  * Built using and for Visual Studio 2017 with prebuilt OpenSSL
+  * `rippled_deps.protoc.2.5.1.zip`
```
Comment (mellery451) [inline appveyor/README.md]: pretty sure we don't need protoc anymore - we build a newer version anyhow as an EP in the cmake file

**Review by mellery451 — APPROVED** [review]

File: appveyor/README.md (hunk 1 above)
Comment (ximinez) [reply appveyor/README.md]: That's easy enough to test!

File: appveyor/README.md [hunk 2]
```diff
@@ -44,3 +16,18 @@ optimize downloads and caching for Appveyor.com CI builds.
...
+      * protoc 2.5.1
+  * `rippled_deps.openssl.1.0.2j.zip`
+    * Includes:
+      * OpenSSL 1.0.2j (64-bit) from http://slproweb.com/products/Win32OpenSSL.html
+  * `rippled_deps.boost.1.70.zip`
+    * Includes:
+      * Boost 1.70, except for the built library binaries in `stage`
+  * `rippled_deps.boost.stage.1.70.zip
+    * Includes:
+      * Boost 1.70, only the built library binaries in `stage`
```
Comment (miguelportilla) [inline appveyor/README.md:24]: Previous archives used the directory name `stage64` vs `stage`. Likely doesn't matter or an easy path fix but just thought I'd mention it.

**Comment by miguelportilla** [comment]
Do we need to keep `rippled_deps17.03.zip`? Appveyor can't use it with tip of develop.

PR changes look good, thanks for updating this repo!

**Review by miguelportilla — APPROVED** [review]

File: appveyor/README.md (hunk 1 above)
Comment (ximinez) [reply appveyor/README.md]: With the PROTOC dependency removed from the script, and the cache invalidated, Appveyor timed out, but not until after the build was done, so this is safe to remove.

File: appveyor/README.md (hunk 2 above)
Comment (ximinez) [reply appveyor/README.md:24]: It doesn't seem to matter. Appveyor works fine either way. If you think it's important enough, I'll rename.

**Comment by ximinez** [comment]
@mickey I'll go ahead and remove `03`, too. I want to keep `04` around for a little while longer so we can still do 1.2.* builds if necessary.

**Pull request merged**